### ISSUE TITLE
Guppy server initialization failed 

### ISSUE THREAD

**Issue by ChloeDG** [nanoporetech/megalodon#42]
Hi there, 

I am trying to call for mod bases 5mC and 5hmC and I am having trouble initiating guppy.
I used the following command: 
`megalodon /home/fast5_pass --guppy-params "-d ./rerio/basecall_models/"  --guppy-config res_dna_r941_min_modbases_5mC_5hmC_CpG_v001.cfg --guppy-server-path /home/hh/Software/ont-guppy-cpu/bin/guppy_basecall_server --mod-motif mh CG 0`

And it fails to initiate guppy. 
```
  File "/home/hh/miniconda3/lib/python3.7/site-packages/megalodon/backends.py", line 357, in start_guppy_server
    'Guppy server initialization failed. See guppy logs ' +
megalodon.megalodon_helper.MegaError: Guppy server initialization failed. See guppy logs in --output-directory for more details.
```
Unfortunately the error logs are not so helpful with the log informing me that 

```
[guppy/error] main: Layer type 'GlobalNormTwoStateCatMod' mustbe last layer.
[guppy/warning] main: An error occurred initialising the basecaller. Aborting.
```
Not sure what I am missing? I tried re-installing guppy and rerio but no change. Any help/ info on calling for 5mC and 5hmC would be amazing. 

Guppy basecaller Version 4.0.14+8d3226e
Ubuntu 16.04

**Comment by marcus1487** [comment]
Was the download of this model from Rerio successful?

The other point here might be that guppy 4.0 has only limited support at the moment ([see here](https://github.com/nanoporetech/megalodon#guppy-40-support)). Could you try to run against guppy 3.6.1?

**Comment by ChloeDG** [comment]
Yes I downloaded the models from rerio successfully. I can try and download guppy version 3.6.1 and see.

**Comment by ChloeDG** [comment]
I just installed guppy version 3.6.1 and I have the exact same error

```
2020-07-21 15:58:31.623148 [guppy/error] Layer type 'GlobalNormTwoStateCatMod' mustbe last layer.
2020-07-21 15:58:31.623208 [guppy/warning] An error occurred initialising the basecaller. Aborting.
```

**Comment by ChloeDG** [comment]
Apologies, I had the wrong path to the guppy_basecall_server. Working well now!!

**Comment by marcus1487** [comment]
Was this resolved with the switch to guppy 3.6.1?

**Comment by ChloeDG** [comment]
I had the wrong path to guppy_basecall_server from the beginning (I had the path to guppy_basecall_server within MinKnow software). So I can not be sure if I would have encountered the same problem with Guppy version 4.1. Either way it is working well with guppy version 3.6.1 now. Once again sorry for that!

**Comment by marcus1487** [comment]
The error messages for this error type should be resolved in the 2.2 release.
